src/openms/include/OpenMS/ANALYSIS/QUANTITATION/AbsoluteQuantitationMethod.h

```diff
@@ -133,12 +130,12 @@ namespace OpenMS
     String component_name_; ///< id of the component
     String feature_name_; ///< name of the feature (i.e., peak_apex_int or peak_area)
     String IS_name_; ///< the internal standard (IS) name for the transition
-    double llod_; ///< lower limit of detection (LLOD) of the transition
-    double ulod_; ///< upper limit of detection (ULOD) of the transition
-    double lloq_; ///< lower limit of quantitation (LLOQ) of the transition
-    double uloq_; ///< upper limit of quantitation (ULOQ) of the transition
-    Int n_points_; ///< number of points used in a calibration curve
-    double correlation_coefficient_; ///< the Pearson R value for the correlation coefficient of the calibration curve
+    double llod_ { 0.0 }; ///< lower limit of detection (LLOD) of the transition
+    double ulod_ { 0.0 }; ///< upper limit of detection (ULOD) of the transition
+    double lloq_ { 0.0 }; ///< lower limit of quantitation (LLOQ) of the transition
+    double uloq_ { 0.0 }; ///< upper limit of quantitation (ULOQ) of the transition
+    Int n_points_ { 0 }; ///< number of points used in a calibration curve
```

Comment:
maybe order members by size?! `n_points_` incurs 4 bytes of padding to make the following double align on 8 byte boundaries. Reordering might not prevent that in this instance because there is only a single integer, but its good practice :)